fearless/Modules/Staking/StakingPayoutConfirmation/StakingPayoutConfirmationInteractor.swift

```diff
@@ -1,7 +1,126 @@
-import UIKit
+import Foundation
+import SoraKeystore
+import CommonWallet
+import RobinHood
+import IrohaCrypto
 
 final class StakingPayoutConfirmationInteractor {
+    let extrinsicService: ExtrinsicServiceProtocol
+    let signer: SigningWrapperProtocol
+    let balanceProvider: AnyDataProvider<DecodedAccountInfo>
```

Comment:
I think price provider is missing

Reply:
Missed it somewhere between branches. Restored